packages/shared/src/schemas.ts

```diff
@@ -13,5 +11,15 @@ const isAddress = (address: string) => {
 
 export const EthAddressSchema = z
   .string()
-  .regex(ETH_ADDRESS_REGEXP)
```

Comment:
This is from @cygnusv 's PR (https://github.com/nucypher/taco-web/pull/598) which my PR is based over.

That being said, I believe it was removed because the `isAddress` refinement check below made using the regex superfluous (cc @cygnusv ).

Reply:
ahh ok! that makes sense :)

Just trying to catch up a bit!